library/lib.ml

```diff
@@ -432,11 +432,6 @@ let mp_of_global = let open GlobRef in function
   | IndRef ind -> Names.Ind.modpath ind
   | ConstructRef constr -> Names.Construct.modpath constr
 
-let rec dp_of_mp = function
-  |Names.MPfile dp -> dp
-  |Names.MPbound _ -> library_dp ()
```

Comment:
This is different from ModPath.dp but probably nobody relied on this difference?

Reply:
AFAICT when we construct `MBId.t`, we put the library dirpath there anyway.